app/controllers/account/billing-info/invoices/list.js

```diff
@@ -0,0 +1,193 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
+
+export default class extends Controller.extend(EmberTableControllerMixin) {
+
+  @computed()
+  get upcomingInvoiceColumns() {
```

Comment:
These are arrays with plain objects. Can be a normal property instead of a computed property

Reply:
@niranjan94 Replaced these with POJOs.

Reply:
@niranjan94 Replaced these with POJOs. Please have a look.